fastar/test_util.py

```diff
@@ -27,8 +27,11 @@ def check(thunk, atol=None, rtol=None):
   jtu.check_close(result, expected, atol, rtol)
 
 def random_box(rng, shape):
-  box_start = rng.randint(shape)
-  box_shape = rng.randint(1, np.subtract(shape, box_start - 1))
+  if np.any(np.less(shape, 0)):
+    raise ValueError
+  box_start = np.array([0 if s == 0 else rng.randint(s) for s in shape])
+  box_shape = np.array([0 if s == 0 else rng.randint(1, s - (start - 1))
+                        for start, s in zip(box_start, shape)])
```

Comment:
Same for this, you could probably express this in NumPy but I don't mind the explicit loop.

Reply:
`rng.randint(0)` fails, therefore not vectorized